task9/task9/Program.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task9
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            //задание 1
+            Console.WriteLine("Введите значения a и h");
+            double a = double.Parse(Console.ReadLine());
+            double h = double.Parse(Console.ReadLine());
+            Console.WriteLine("Таблица перевода дюймов в см:");
+            for (double number = 1; number <= a; number += h)
+                Console.WriteLine($"{number} дюйм = {number * 2.54} см");
+
+
+            //задание 2
+            Console.ForegroundColor = ConsoleColor.Blue;
+            Console.WriteLine("\nВведите значение n");
+            int n = int.Parse(Console.ReadLine());
+            Console.WriteLine($"Натуральные числа, квадрат которых меньше {n}:");
+            for (int num = 1; Math.Pow(num, 2) < n; num++)
```

Comment:
Квадрат числа лучше вычислять умножением
```suggestion
            for (int num = 1; num * num < n; num++)
```